### PR TITLE
Bug 1219655 - Statusbar tests for FTU + helpers

### PR DESCRIPTION
**Pull request by sfoster**
Adds in the remaining statusbar test from test_ftu_status_bar.py and some helpers for Ftu.

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=024fcf1aba7b602bca1c92b7c6dcb53b4da18d9a

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=024fcf1aba7b602bca1c92b7c6dcb53b4da18d9a

File: apps/ftu/test/marionette/lib/ftu.js [hunk 1]
```diff
@@ -78,6 +103,63 @@ Ftu.prototype = {
...
+    });
+    return hash;
+  },
+
+  goNext: function() {
+    var loadingOverlay = this.client.findElement('#loading-overlay');
+    var finishScreen = this.client.findElement(Ftu.Selectors.finishScreen);
+    this.client.waitFor(function() {
+      return !loadingOverlay.displayed();
+    });
```
Comment (mikehenrty) [inline apps/ftu/test/marionette/lib/ftu.js]: just fyi there is also
```
this.client.helper.waitForElementToDisappear('#loading-overlay');
```

File: apps/ftu/test/marionette/lib/ftu.js [hunk 2]
```diff
@@ -78,6 +103,63 @@ Ftu.prototype = {
...
+      return button.enabled;
+    });
+    var currentHash = this.getLocationHash();
+    button.tap();
+    this.client.waitFor(function() {
+      return (
+        finishScreen.displayed() ||
+        currentHash !== this.getLocationHash()
+      );
+    }.bind(this), { timeout: 2000 });
```
Comment (mikehenrty) [inline apps/ftu/test/marionette/lib/ftu.js]: why explicitly set a timeout here?

Comment (sfoster) [inline apps/ftu/test/marionette/lib/ftu.js]: Some of these steps can take a while, I wanted a bigger timeout

File: apps/ftu/test/marionette/lib/ftu.js (hunk 1 above)
Comment (sfoster) [inline apps/ftu/test/marionette/lib/ftu.js]: I need to find the source for this, does it return immediately if its already not displayed? Or rely on a transition/animation

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d83bf4bf886086e787a9da7d1664f23a39e224c5

**Pull request merged**